packages/pricing/src/models/currency.ts

```diff
@@ -14,14 +14,10 @@ class Currency {
   @Property({ columnType: "text" })
   name: string
 
-  // @Property({ persist: false })
-  // get includes_tax() {
-  //   // TODO: This comes from a feature flag
-  //   // Figure out how we're handling FF in modules
-  //   // For now, returning default as true
-  //   // This should also not fall on the hands of the model
-  //   return true
-  // }
+  // TODO: This is added here to make the schema compatible
+  // with the core. Remove this when core is no longer running migrations
+  @Property({ columnType: "boolean", nullable: true })
+  includes_tax?: boolean
```

Comment:
suggestion: If the module is shared and use the core connection, then the migration is managed by the core and the module should not run its migration. it should run its migrations only if it is isolated otherwise we rely on the core. wdyt? In that case we could remove this from here and the migration

Reply:
Not a big fan of that idea. The model and schema can easily be corrupted by one another. My ideal scenario would actually be to scope the tables by the name of the module. In a world of modules, I think this would actually be a good thing. 

Additionally, we're making an assumption that the core models will be exactly the same as the module models. Which isn't the case with pricing models. There are other migrations that need to run.

Reply:
Sorry i thought i did remove this comment 🤣 but i wanted to start a discussion on how we would handle this. Cause another problem could be that when using the module the user might now expect to loose his current data in the core, so i think it is a larger topic that needs to be discussed